### PR TITLE
Update import-external-messages-to-teams.md

### PR DESCRIPTION
**Pull request by RamjotSingh**

### CODE REVIEW

File: msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md [hunk 1]
```diff
@@ -105,7 +102,7 @@ Creating a channel for the imported messages is similar to the create team scena
 #### Request (create a channel in migration state)
 
 ```http
-POST https://graph.microsoft.com/beta/teams/{id}/channels
+POST https://graph.microsoft.com/v1.0/teams/{id}/channels
```
Comment (AkJo) [inline msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md:105]: {team-id}

File: msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md [hunk 2]
```diff
@@ -156,7 +153,7 @@ After the team and channel have been created, you can begin sending back-in-time
 #### Request (POST message that is text-only)
 
 ```http
-POST https://graph.microsoft.com/beta/teams/teamId/channels/channelId/messages
+POST https://graph.microsoft.com/v1.0/teams/teamId/channels/channelId/messages
```
Comment (AkJo) [inline msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md:156]: {team-id}

Comment (AkJo) [inline msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md:156]: {channel-id}

File: msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md [hunk 3]
```diff
@@ -180,7 +177,7 @@ POST https://graph.microsoft.com/beta/teams/teamId/channels/channelId/messages
 HTTP/1.1 200 OK
 
 {
-   "@odata.context":"https://graph.microsoft.com/beta/$metadata#teams('teamId')/channels('channelId')/messages/$entity",
+   "@odata.context":"https://graph.microsoft.com/v1.0/$metadata#teams('teamId')/channels('channelId')/messages/$entity",
```
Comment (AkJo) [inline msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md:180]: If you are using placeholder values, then use the format 'team-id' and 'channel-id'. If you are putting down illustrative values, then put a real team id and channel id in here and elsewhere.

File: msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md [hunk 4]
```diff
@@ -227,7 +224,7 @@ HTTP/1.1 200 OK
 > **Note**: There are no special permission scopes in this scenario since the request is part of chatMessage; scopes for chatMessage apply here as well.
 
 ```http
-POST https://graph.microsoft.com/beta/teams/teamId/channels/channelId/messages
+POST https://graph.microsoft.com/v1.0/teams/teamId/channels/channelId/messages
```
Comment (AkJo) [inline msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md:227]: Same feedback.

File: msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md [hunk 5]
```diff
@@ -250,7 +247,7 @@ POST https://graph.microsoft.com/beta/teams/teamId/channels/channelId/messages
 HTTP/1.1 200 OK
 
 {
-    "@odata.context": "https://graph.microsoft.com/beta/$metadata#teams('teamId')/channels('channelId')/messages/$entity",
+    "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#teams('teamId')/channels('channelId')/messages/$entity",
```
Comment (AkJo) [inline msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md:250]: Same feedback.

File: msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md [hunk 6]
```diff
@@ -285,12 +282,12 @@ HTTP/1.1 200 OK
...
 
-Once the message migration process has completed, both the team and channel are taken out of migration mode using the  `completeMigration`  method. This step opens the team and channel resources for general use by team members. The action is bound to the `team` instance.
+Once the message migration process has completed, both the team and channel are taken out of migration mode using the  `completeMigration`  method. This step opens the team and channel resources for general use by team members. The action is bound to the `team` instance. All channels must be completed out of migration mode before the team can be completed.
 
-#### Request (end team migration mode)
+#### Request (end channel migration mode)
 
 ```http
-POST https://graph.microsoft.com/beta/teams/teamId/completeMigration
+POST https://graph.microsoft.com/v1.0/teams/teamId/channels/channelId/completeMigration
```
Comment (AkJo) [inline msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md:290]: Same feedback.

File: msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md [hunk 7]
```diff
@@ -300,10 +297,10 @@ POST https://graph.microsoft.com/beta/teams/teamId/completeMigration
 HTTP/1.1 204 NoContent
 ```
 
-#### Request (end channel migration mode)
+#### Request (end team migration mode)
 
 ```http
-POST https://graph.microsoft.com/beta/teams/teamId/channels/channelId/completeMigration
+POST https://graph.microsoft.com/v1.0/teams/teamId/completeMigration
```
Comment (AkJo) [inline msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md:303]: {team-id}

File: msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md [hunk 8]
```diff
@@ -373,7 +362,6 @@ HTTP/1.1 204 No Content
 |Messages with rich text|Videos|
 |Message reply chain|Announcements|
 |High throughput processing|Code snippets|
-||Adaptive cards|
```
Comment (AkJo) [inline msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md:376]: Was this intentional? We are supporting adaptive cards?

**Review by AkJo — APPROVED** [review]

File: msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md (hunk 8 above)
Comment (RamjotSingh) [reply msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md:376]: Yes

**Comment by KirtiPereira** [comment]
@AkJo - please approve the PR for us to take it up for merging.
Thank you.

**Review by KirtiPereira — APPROVED** [review]
Approved

**Pull request merged**